### PR TITLE
MACRO: IS_x updated

### PR DESCRIPTION
**Pull request by Dorbedo**
updated the "IS_x" marcos because of the new "isEqualType" command

### CODE REVIEW

**Comment by Dorbedo** [comment]
I have choosen "IsEqualType" because it was slightly faster than "IsEqualTo".

File: addons/main/script_macros_common.hpp [hunk 1]
```diff
@@ -829,21 +829,21 @@ Macros: IS_x()
...
+#define IS_DISPLAY(VAR)  IS_META_SYS(VAR,displayNull)
+#define IS_GROUP(VAR)    IS_META_SYS(VAR,grpNull)
+#define IS_OBJECT(VAR)   IS_META_SYS(VAR,objNull)
+#define IS_SCALAR(VAR)   IS_META_SYS(VAR,0)
+#define IS_SCRIPT(VAR)   IS_META_SYS(VAR,scriptNull)
+#define IS_SIDE(VAR)     IS_META_SYS(VAR,west)
 #define IS_STRING(VAR)   IS_META_SYS(VAR,"STRING")
-#define IS_TEXT(VAR)     IS_META_SYS(VAR,"TEXT")
-#define IS_LOCATION(VAR) IS_META_SYS(VAR,"LOCATION")
+#define IS_TEXT(VAR)     IS_META_SYS(VAR,parseText "--</br>--")
```
Comment (commy2) [inline addons/main/script_macros_common.hpp]: Should be:
`IS_META_SYS(VAR,text "")`
imo, which is ~3 times faster.

Comment (Dorbedo) [inline addons/main/script_macros_common.hpp]: thx, I have changed it.

**Comment by commy2** [comment]
:+1:

**Pull request merged**